hack/run.sh

```diff
@@ -96,6 +95,8 @@ elif [[ "${action}" == "benchmark-filter" ]]; then
   "${ROOT_DIR}/data-plane/benchmarks/run.sh" "$2"
 elif [[ "${action}" == "benchmark-filters" ]]; then
   "${ROOT_DIR}/data-plane/benchmarks/run.sh"
+elif [[ "${action}" == "./hack/update-codegen.sh" || "${action}" == "./hack/run.sh format-java" ]]; then
```

Comment:
```suggestion
elif [[ "${action}" == "format-java" ]]; then
```
Here, `update-codegen.sh` is an already existing script, so we don't need to check for it.

Also, `${action}` is the argument passed to the script, so it would be just `format-java`

Reply:
Should I make it like this?
```
elif [[ "${action}" == "format-java" ]]; then
  cd "${ROOT_DIR}/data-plane"
  ./mvnw spotless: apply
```

Reply:
Almost! There can't be a space between `spotless:` and `apply`, so I would do:
```bash
elif [[ "${action}" == "format-java" ]]; then
  cd "${ROOT_DIR}/data-plane"
  ./mvnw spotless:apply
```

Reply:
Ok thank you